src/renderer/services/synchronizer/wallets.js

```diff
@@ -172,7 +172,13 @@ class Action {
   }
 
   findLatestTransaction (transactions) {
-    return last(sortBy(transactions, 'timestamp'))
+    var latestTransaction = transactions[0]
+
+    for (const i in transactions) {
+      if (transactions[i].timestamp > latestTransaction.timestamp) latestTransaction = transactions[i]
```

Comment:
can you use curly brackets for if statements please, as it makes it more consistent across the codebase. ones without brackets are harder to read imo